scenarioo-client/app/step/index.ts

```diff
@@ -5,6 +5,10 @@ import './screenAnnotations/screenAnnotationsButton.directive';
 import './screenAnnotations/screenAnnotationInfoPopup.controller';
 import './screenAnnotations/screenAnnotations.service';
 import './stepNotFound/stepNotFound.component';
+import './fallback/fallback.component';
+import './navigation/navigation.component';
```

Comment:
navigation might be duplicated. What about StepNavigation?

Reply:
Good point. I renamed the component and html to more clearly differentiate it from the shared Naviation.